go.mod

```diff
@@ -0,0 +1,48 @@
+module github.com/giantswarm/coredns-app
+
+go 1.13
+
+require (
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // indirect
```

Comment:
Not blocking, just wondering where these imports come from. Seems weird to have such an indirect.